deployment/ansible/roles/k8s/tasks/components/kuma.yml

```diff
@@ -0,0 +1,102 @@
+---
+- name: Install argocd
+  kubernetes.core.helm:
+    chart_repo_url: https://dirsigler.github.io/uptime-kuma-helm
+    chart_ref: uptime-kuma
+    chart_version: 2.5.4
+    release_name: uptime-kuma
+    namespace: monitoring
+    create_namespace: yes
+- name: Make shure kuma has an ingress
+  kubernetes.core.k8s:
+    state: present
+    namespace: monitoring
+    definition:
+      apiVersion: traefik.containo.us/v1alpha1
+      kind: IngressRoute
+      metadata:
+        name: kuma-uptime
+        namespace: monitoring
+      spec:
+        entryPoints:
+          - websecure
+        routes:
+          # temporary workaround for traefik bug (interoperability with ingress)
+          - kind: Rule
+            match: Host(`uptime.nav.tum.sexy`) && PathPrefix(`/.well-known/acme-challenge/`)
+            priority: 13
+            services:
+              - name: cm-acme-http-solver-mppjd
+                port: 8089
+                kind: Service
+          - kind: Rule
+            match: Host(`uptime.nav.tum.sexy`)
+            priority: 12
+            services:
+              - name: uptime-kuma
+                port: 3001
```

Comment:
Since it's still using the old domain, is this workaround still needed?

Reply:
We do not have the ability to spwan `*.nav.tum.de` domains, as they did not grant this request.
Any staging or admin activity (jaeger, argocd, traefik, uptime) will have to use `*.nav.tum.sexy`